src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -833,6 +839,33 @@ class HookUtils(object):
                            'Error removing file: %s' % (filename))
         return True
 
+    def _execjob_abort_handler(self, event, cgroup, jobutil):
+        """
+        Handler for execjob_abort events.
+        """
+        pbs.logmsg(pbs.EVENT_DEBUG4, '%s: Method called' % caller_name())
```

Comment:
I can't comment on areas you haven't changed, so I'll do so here. Suggest you add another log message to _execjob_end_handler() indicating the event for which it has been called.

Reply:
ok.

Reply:
Actually, we don't need to add new log message showing the actual hook event service as 2 log messages would automatically appear before and after the execjob_end_handler() is invoked:

02/20/2019 14:22:59;0800;pbs_python;Hook;pbs_python;main: Event type is execjob_abort
...
02/20/2019 14:22:59;0800;pbs_python;Hook;pbs_python;_execjob_abort_handler: Method called
NOTE: With the fix, "execjob_abort_handler" would now be "execjob_end_handler"
...
02/20/2019 14:22:59;0800;pbs_python;Hook;pbs_python;main: Hook handler returned success for execjob_abort event

Reply:
You are correct. That makes it pretty apparent which event triggered it.